GBA/Display.zig

```diff
@@ -0,0 +1,122 @@
+const GBA = @import("core.zig").GBA;
+const IO = @import("IO.zig");
+const Color = @import("color.zig").Color;
+
+const VRAM_BASE_ADDR = 0x06000000;
+const SPRITE_VRAM = VRAM_BASE_ADDR + 0x10000;
+const MODE_4_PAGE_SIZE = 0xA000;
+var current_page = VRAM_BASE_ADDR;
+
+/// Controls the capabilities of background layers
+///
+/// Modes 0-2 are tile modes, modes 3-5 are bitmap modes
+pub const DisplayMode = enum(u3) {
+    /// Tiled mode
+    ///
+    /// Provides 4 normal background layers (0-3)
+    Mode0,
```

Comment:
While you are moving stuff around, you should transform all the enum values to snake_case to fit with the Zig standard coding convention. This was my first Zig project and I just did my C++ coding style, for my other projects I switched to the Zig conding convention but I never bothered with ZigGBA

Reply:
Yeah, although the lower_snake_case enums are one thing I've never completely sold myself on in zig.